lapce-ui/src/editor/tab_header_content.rs

```diff
@@ -384,7 +492,19 @@ impl Widget<LapceTabData> for LapceEditorTabHeaderContent {
                 )
                 .build()
                 .unwrap();
-            let text_size = text_layout.size();
+            let path_layout = ctx
```

Comment:
I don't see clearly on Github's UI, but you might want to check if `file_path` is empty here and skip this whole thing in that case. `TabRect` could have an optional `path_layout` to make it work.